Assets/AnkrSDK/Runtime/MirageAPI/MirageID/Helpers/MirageIdEnv.cs

```diff
@@ -0,0 +1,65 @@
+using System.Collections.Generic;
+using AnkrSDK.MirageAPI.MirageID.Implementation;
+
+namespace AnkrSDK.MirageAPI.MirageID.Helpers
+{
+	public class MirageIdEnv : IMirageIdEnv
+	{
+		private static readonly Dictionary<Environments, EnvSetting> Environments = CreateEnvironments();
+
+		private string _currentEnvMirageId;
```

Comment:
you don't have to store each string separately - change it to get only the property and get those strings from cached envSettings